### DISCUSSION TITLE
diskspace not freeing up

### DISCUSSION THREAD

**Discussion by d3athg0d** [telekom-security/tpotce#1035] [Q&A] [answered]
Hi,

i have read through the discussions and wiki but not able to get it right.
probably i have misunderstood but can some kind souls point me in the right directions

currently i am facing with diskspace not being free up issue.
i have set the logrotate as indicated in the wiki
i have try to delete the indices in the Index Management

but the diskspace just do not free up or go down
.....
![Screenshot 2022-03-07 211724](https://user-images.githubusercontent.com/78822442/157042344-dd155a0a-1008-4c4c-90ff-bdac77c4e118.png)
![Screenshot 2022-03-07 211706](https://user-images.githubusercontent.com/78822442/157042353-564c99b9-a9be-4d89-a23d-92c0d43a20a0.png)
![Screenshot 2022-03-07 221800](https://user-images.githubusercontent.com/78822442/157051374-d5f4ce1e-dc84-4f54-8d51-f56f5a8a0b76.png)

**Comment by t3chn0m4g3** [comment, marked as answer]
#1023 should have you covered.
